maven/plugins/nodejs-maven-plugin/changes.xml

```diff
@@ -22,7 +22,11 @@
 <document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/changes/1.0.0"
     xsi:schemaLocation="http://maven.apache.org/changes/1.0.0 http://maven.apache.org/plugins/maven-changes-plugin/xsd/changes-1.0.0.xsd">
   <body>
-
+    <release version="3.1.0">
```

Comment:
i would rather make this version "3.0.2" and the action type to "update". it's not really adding a new feature, just extending some checks.

Reply:
I've changed it to 3.0.1, as this would be the next in line.